packages/background/src/controllers/NetworkController.ts

```diff
@@ -191,8 +196,7 @@ export default class NetworkController extends BaseController<NetworkControllerS
             newNetworks[networkKey].order = order;
         });
 
-        this.networks = newNetworks;
-        this.orderNetworks();
+        this.networks = normalizeNetworksOrder(newNetworks);
```

Comment:
You're already doing it in the network setter. So, in this case, there isn't necessary to invoke `normalizeNetworksOrder`.

When you do:

```javascript
this.networks = newNetworks
```
The code that is being invoked is the setter:

```typescript
public set networks(networks: Networks) {
        this.store.updateState({
            availableNetworks: normalizeNetworksOrder(networks),
        });
    }
```